packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -613,7 +618,7 @@ const downloadConfiguredModel = async () => {
 };
 
 const createConfiguration = async (force: boolean = false) => {
-	if (!model.value) return;
+	if (!model.value || !props.node.state?.transientModelConfig?.name) return;
```

Comment:
Is this test the same than with `isSaveDisabled`?
If yed, why not use it to return early?